src/sentry/conf/server.py

```diff
@@ -2490,7 +2492,13 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 STATUS_PAGE_ID: str | None = None
 STATUS_PAGE_API_HOST = "statuspage.io"
 
-SENTRY_SELF_HOSTED = True
+# This supersedes SENTRY_SINGLE_TENANT and SENTRY_SELF_HOSTED.
+# An enum is better because there shouldn't be multiple "modes".
+SENTRY_MODE = SentryMode.SELF_HOSTED
+
+# For compatibility only. Prefer using SENTRY_MODE.
+SENTRY_SELF_HOSTED = SENTRY_MODE == SentryMode.SELF_HOSTED
```

Comment:
Should this all be in this PR, seems like something that might be worth putting in it's own PR so we can discuss this approach independently of this robots txt stuff

Reply:
> i'm gonna separate SENTRY_MODE after i figure out how to send that to frontend btw